manager2/src/app/admin/projects/projects.component.ts

```diff
@@ -270,4 +306,28 @@ export class ProjectsComponent implements OnInit {
         }
         return res;
     }
+
+    accept_project(project) {
```

Comment:
3 methods to take a decision for the pending project:
accept -> straight to backend, no path or additional info added. to be filled later from the existing project management page.
modify -> fills this.new project and admin can use the legacy way of accepting the project, and can modify or fill in the missing info.
reject: straight to backend, pending project is rejected